app/services/ad_reminder.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+# A class to implement all logic related to sending of reminder e-mails
+class AdReminder
+  include Rails.application.routes.url_helpers
+  default_url_options[:host] = Rails.application.config.web_host
+  default_url_options[:protocol] = Rails.application.config.web_protocol
```

Comment:
Can this be configured like `Rails.application.routes.default_url_options[:host] = "www.potres-petrinja.hr"`?

Reply:
I was thinking something like this as well. Sorry for the confusion.

We could also store that domain in the ENV var so we don't have it in several places across the app code.

Reply:
Vlado's suggestion works and I went with that. I didn't go with ENV because we don't have a ENV management set up for local development. If we set that up I'm in favour or extracting it into an ENV var.

Reply:
I didn't know you can do `routes.default_url_options`! Looks good! Thanks!